app/controllers/dances_controller.rb

```diff
@@ -69,6 +69,10 @@ def collections
     @meter_collection = Meter.all
   end
 
+  def dances
+    Dance.all
```

Comment:
For future reference, an optimization technique you could implement here:

```
def dances
    @dances ||= Dance.all
    # equivelent to @dances = @dances || Dances.all
end
```

This caches the variable's value in memory. You are essentially telling ruby "if the variable is defined, don't redefine it," preventing the same DB transaction from happening twice if the `dances` method is called more than once in a request.